src/tox_lsr/test_scripts/runansible-doc.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# A shell wrapper around ansible-doc.
+
+set -u
```

Comment:
you can't use `set -euo pipefail` here?

Reply:
If we do that, the collection test stops there. (;_;)

Reply:
Where?  I think if this is rewritten slightly:
```bash
if ! ANSIBLE_COLLECTIONS_PATHS="$MY_LSR_TOX_ENV_DIR" ansible-doc -M "$LSR_ROLE2COLL_MODULES" \
      -vvv --type module "$module"; then
    rval=1
fi
```
I think it was the `expr` - if `expr` evaluates to `0`, then it returns an error to the shell.  But I don't think the particular error returned from ansible-doc is significant - all you want to do is exit `1` if there is an error, or `0` if no errors.

If you really want to preserve the code as is, but use `set -euo pipefail`, try this:
`    rval=$( expr $rval + $r ) || :`